.gitignore

```diff
@@ -10,3 +10,6 @@ dist/*
 yarn.lock
 package-lock.json
 npm-shrinkwrap.json
+
+# IDE related
+.idea
```

Comment:
revert this; your IDE files belong in your personal global gitignore, not littered in every project you touch